manifests/agent/install.pp

```diff
@@ -1,8 +1,7 @@
 # Install the puppet client installation
 class puppet::agent::install {
-
-  package { $puppet::params::client_package:
-    ensure => $::puppet::version,
+  package { $puppet::client_package:
```

Comment:
BTW @mmoll, sorry if that came across crappy... I just reread that and realized I could have worded it better.